core/tests/data/exploration_dict.js

```diff
@@ -0,0 +1,501 @@
+var explorationDict = {
```

Comment:
You should not include this globally. It then becomes present in all tests, and has the possibility of colliding with other variables named explorationDict.

Instead, either declare it within the test itself, or have the specific test load a json file or something. It needs to be kept to the specific test.

Reply:
Done!